modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -681,10 +681,10 @@
                 var manipulationType = $scope.type.toLowerCase();
                 $scope.manipulationTypes = [];
                 if(['unicode', 'string'].indexOf(manipulationType) > -1) {
-                    $scope.manipulationTypes = ['toUpper', 'toLower', 'capitalize', 'URLEncode', 'join', 'split', 'substring', 'format', 'parseDate'];
+                    $scope.manipulationTypes = ['toUpper', 'toLower', 'capitalizeFirstLetter', 'URLEncode', 'join', 'split', 'substring', 'format', 'parseDate'];
                 }
                 if('date' === manipulationType) {
-                    $scope.manipulationTypes.push('dateTime');
+                    $scope.manipulationTypes.push('date.time');
```

Comment:
@MichalKus It seems to me that changing this may break using those manipulations in Tasks (since those keys are used in KeyEvaluator class). Did you verify that this works?